### DISCUSSION TITLE
¿Whatssaap o Telegram?

### DISCUSSION THREAD

**Discussion by migueel17** [Launch-X-Latam/MisionFrontEnd#53] [General]
Cren que los que no estamos muy familiarisados, se les facilitaria crear un grupo oficial sobre el curso,?
 o creen que es mejor adaptarse al sitio de github aun que cueste mas trabajo jaja.

**Comment by LaMazorca** [comment]
Según el programa está prohibido tener grupos ajenos al MS teams y al git, además si nos vamos a dedicar al desarrollo saber utilizar el git, es lo mejor que podemos hacer

**Comment by Emilio17MC** [comment]
De acuerdo con el programa LaunchX, no está permitido realizar ni utilizar canales de comunicación externos, por eso es importante adaptarse a GitHub.

**Comment by migueel17** [comment]
vale esta bien solo tenia esa duda, igual gracias.

**Comment by gjordanra** [comment]
estoy de acuerdo hay que acostumbrarse a usar los canales normales de desarrollo, github es el más común

**Reply by OATS74** [reply]
Eso está prohibido amigo. Y el castigo es expulsión del proyecto

**Reply by gjordanra** [reply]
@OATS74 no te entiendo de que hablas, se supone que debemos usar Github para comunicarnos y colaborar, eso es lo que estamos asiendo en este mismo chat, no Whatsapp, ;-)

**Reply by egahdzv** [reply]
si es lo mejor todo por las herramientas oficiales.

**Comment by egahdzv** [comment]
Creo todo es digerible, en caso de que no te familiarices o se te dificulte es mejor salir no pasa nada. a mi consideración no quiero ofender a nadie, creo que la seguridad de todos es lo mejor sin filtrar números o nombre ok. sol como dice es Agente es leer y estar atentos.
